tests/summary_fields.feature

```diff
@@ -56,4 +56,30 @@ Feature: Summary fields
       | dvlad7909   |
       | dmitriivlad |
       # | ab054       |
-      |stas00       |
\ No newline at end of file
+      |stas00       |
+
+  Scenario Outline: Verify number of gists UI vs API
+    When UI: Input <user_name> into search field
+    And UI: Click Search button
+    And API: send get request to <user_name>
+    And API: verify status code is 200
+    Then Verify gists field values
+
+    Examples:
+      | user_name   |
+      | dvlad7909   |
+      | dmitriivlad |
+      # | ab054       |
+      |stas00       |
+
+  Scenario: Verify gists data is updated on page refresh
+    When UI: Input dvlad7909 into search field
+    And UI: Click Search button
+    And API: send get request to dvlad7909
+    And API: verify status code is 200
+    And API: send post request to dvlad7909 to add 1 more gist
```

Comment:
That's cool that you have this level of testing

Reply:
Yea, but I didn't found API for followers/following functionality. So that tests left for manual testing in backlog folder.